scripts/release-notes.py

```diff
@@ -378,33 +378,35 @@ def spin():
 print("summary: Additions and changes in CockroachDB version", current_version, "since version", previous_version)
 print("---")
 print()
+print("## Month Day, Year")
```

Comment:
Did you mean to commit this?

Reply:
Yes, the output was missing the h2 header at the top of the page. But I'll more clearly indicate that `Month Day, Year` are placeholders.